### PR TITLE
fix(stdlib): pagerduty tests did not include _value column

### PR DESCRIPTION
**Pull request by jsternberg**
### Done checklist
- [x] docs/SPEC.md updated
- [x] Test cases written

### CODE REVIEW

File: stdlib/pagerduty/pagerduty_test.flux [hunk 1]
```diff
@@ -4,24 +4,24 @@ import "array"
 import "pagerduty"
 import "testing"
 
-input = array.from(
+input = () => array.from(
```
Comment (nathanielc) [inline stdlib/pagerduty/pagerduty_test.flux:7]: why the change to making it a function? Does it have any effect on the result?

**Review by nathanielc — APPROVED** [review]
LGTM

File: stdlib/pagerduty/pagerduty_test.flux (hunk 1 above)
Comment (jsternberg) [reply stdlib/pagerduty/pagerduty_test.flux:7]: Forgot to respond to this. No, I thought it did so I changed it and it didn't have an effect. Then I forgot to put it back. This works nicely anyway though.
